coalib/bears/requirements/PackageRequirement.py

```diff
@@ -0,0 +1,43 @@
+from coala_decorators.decorators import generate_eq
+from coala_decorators.decorators import generate_repr
```

Comment:
coala has done this. i ran coala when i had it ``from coala_decorators.decorators import generate_repr, generate_eq`` and it done it this way :( dammit coala, should i file an issue about this, it shouldnt work like this?